test/e2e/alert.test.js

```diff
@@ -10,6 +10,7 @@ describe('vl-alert', async () => {
 
     it('Als ik een alert met title gebruik, wordt de text onder messages gerenderd', async () => {
         const alert = await vlAlertPage.getAlertWithIconAndTitle();
+        await assert.eventually.isOk(alert.hasTitle());
```

Comment:
moeten we ne keer beslissen of we isOk of isTrue gebruiken? wat is het verschil?

Reply:
isOk test voor truthy: mag hier eigenlijk isTrue zijn